### DISCUSSION TITLE
glsp-example repository node-json-vscode example error

### DISCUSSION THREAD

**Discussion by christianmalek** [eclipse-glsp/glsp#997] [Q&A] [answered]
The [glsp-example node-json-vscode example](https://github.com/eclipse-glsp/glsp-examples/tree/master/project-templates/node-json-vscode) doesn't work with Node 18.16.0 under Windows 10. It throws following error if I run `yarn install` on the client directory:

![image](https://github.com/eclipse-glsp/glsp/assets/2873986/5f3b472a-1396-425f-a49b-a7d9d4aa7b8f)

There are various solutions to this problem, such as downgrading to Node 16. Unfortunately, this is not a solution for me.

It seems that this problem is due to the fact that Webpack 4 is still used in the example, which uses MD4 checksums. These are no longer present in the current Node 18 LTS version. An upgrade to Webpack 5 should solve this problem.

**Comment by tortmayr** [comment, marked as answer]
Thanks for raising this issue!. The problem should be resolved once https://github.com/eclipse-glsp/glsp-examples/pull/186 is merged.

**Reply by christianmalek** [reply]
Thanks for the info! I'll try the changes from the PR directly next monday. This helps a lot!

**Reply by christianmalek** [reply]
Build process is now working on my machine!
